functions/helpFunctions.tsx

```diff
@@ -191,3 +192,14 @@ export function chooseCorrelatedAssessment(
   }
   return null;
 }
+
+// check if all scores are set
+export function checkScores(assessments: AssessmentType[]): boolean {
+  let status: boolean = true;
+  assessments.map((assessment) => {
+    if (assessment.score === '') {
+      status = false;
+    }
+  });
```

Comment:
Would it be possible to use the cool `some()` function you used earlier?

Reply:
Niiice, but I think it has to be `!==` because now it will return `true` when it is supposed to be `false`

Reply:
I dont understand, isn't that what we want? That it checks if any of the assessments has `score === ''` ?
I understand it like the map works like a foreach, and if *any* of the elements in assessment has `score === ''`, return false.
So if `some()` returns true if *any* of the elements, so we have to `!` it like you thought of:)

```suggestion
  return !assessments.some((e) => e.score === '');
```

Reply:
However, you code works as it is and we're finished within the week soooo 🤷  :)

Reply:
Ups i'm sorry. I think i just confused myself:) I will commit your suggestion 💯